src/ussd_popi_user_data.js

```diff
@@ -372,39 +372,67 @@ go.app = function() {
                     }
                 },
                 next: function(content) {
-                    // TODO: Update ID on identitystore
-                    return 'state_updated';
+                    return 'state_create_identification_change';
                 }
             });
         });
 
-        self.add('state_change_passport', function(name) {
+        self.add('state_passport_origin', function(name) {
             return new ChoiceState(name, {
                 question: $('What is the country of origin of the passport?'),
                 choices: [
-                    new Choice('state_change_passport_zim', $('Zimbabwe')),
-                    new Choice('state_change_passport_moz', $('Mozambique')),
-                    new Choice('state_change_passport_mal', $('Malawi')),
-                    new Choice('state_change_passport_ng', $('Nigeria')),
-                    new Choice('state_change_passport_drc', $('DRC')),
-                    new Choice('state_change_passport_som', $('Somalia')),
-                    new Choice('state_change_passport_oth', $('Other'))
+                    new Choice('zw', $('Zimbabwe')),
+                    new Choice('mz', $('Mozambique')),
+                    new Choice('mw', $('Malawi')),
+                    new Choice('ng', $('Nigeria')),
+                    new Choice('cd', $('DRC')),
+                    new Choice('so', $('Somalia')),
+                    new Choice('other', $('Other'))
                 ],
                 next: function(choice) {
-                    // TODO: Update passport on identitystore
-                    return 'state_update_passport';
+                    return 'state_passport_no';
```

Comment:
Not strictly necessary, but just for future, `next` can just be a string, so you don't have to have a function that returns the string, you can just have the value as the string: http://vumi-jssandbox-toolkit.readthedocs.io/en/master/states/reference.html#ChoiceState